contracts/ERC4626/VenusERC4626Isolated.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.25;
+
+import { VenusERC4626 } from "./Base/VenusERC4626.sol";
+import { IComptroller } from "./Interfaces/IComptroller.sol";
+import { VToken } from "@venusprotocol/isolated-pools/contracts/VToken.sol";
+import { IERC20Upgradeable, SafeERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import { ensureNonzeroAddress } from "@venusprotocol/solidity-utilities/contracts/validators.sol";
+import { RewardsDistributor } from "@venusprotocol/isolated-pools/contracts/Rewards/RewardsDistributor.sol";
+import { MaxLoopsLimitHelper } from "@venusprotocol/isolated-pools/contracts/MaxLoopsLimitHelper.sol";
+import { IProtocolShareReserve } from "./Interfaces/IProtocolShareReserve.sol";
+
+/// @title VenusERC4626Isolated
+/// @notice ERC4626 wrapper for Venus Isolated Pool vTokens
+contract VenusERC4626Isolated is VenusERC4626, MaxLoopsLimitHelper {
+    uint256 public constant LOOPS_LIMIT = 100;
```

Comment:
In other contracts we pass this value in the initialiser. Maybe we can keep the standard.